src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -152,6 +152,12 @@ namespace AppInstaller::Manifest
                 resultErrors.emplace_back(ManifestError::ExeInstallerMissingSilentSwitches, ValidationError::Level::Warning);
             }
 
+            if (installer.InstallerType == InstallerTypeEnum::Portable &&
+                installer.Commands.size() > 1)

```

Comment:
>Commands [](http://example.com/codeflow?start=26&length=8)

The commands may be populated from root level, we'll have to check only the installer level declares more than 1 or if it's not super important, we could just remove the check

Reply:
I don't think this is super necessary, so I removed this check